plugin.yaml

```diff
@@ -0,0 +1,28 @@
+podSpec:
+  containers: []
+  restartPolicy: Never
+  serviceAccountName: sonobuoy-serviceaccount
+  tolerations:
+  - effect: NoSchedule
+    key: node-role.kubernetes.io/master
+    operator: Exists
+  - key: CriticalAddonsOnly
+    operator: Exists
+  - key: kubernetes.io/e2e-evict-taint-key
+    operator: Exists
+      
+sonobuoy-config:
+  driver: Job
+  plugin-name: linkerd2-conformance
+  result-format: junit
+spec:
+  command:
+  - /bin/bash
+  args:
+      - ./run.sh
```

Comment:
Hmm, Do we need `run.sh` actually? By removing it we can directly pass the planned config yaml here which is confused by a conformance helper and downloads the binary, etc and then calls the `testutil.TestHelper` with relevant configuration.

By removing this we only have a single layer i.e Go Conformance Helpers where we perform book-keeping tasks initially.. I think we can remove it unless there is some use-case. :)

Reply:
It exists only to make sure that the command section is not too untidy and filled with commands. Once we get https://github.com/linkerd/linkerd2/pull/4532 merged, I'll make sure to remove the binary installation process from here and make it a part of the config yaml. :smile: